package.json

```diff
@@ -126,16 +126,19 @@
         "defaults"
     ],
     "scripts": {
-        "black": "black -v .",
-        "black_files": "black -v ",
+        "py-format": "ruff format .",
+        "py-format-files": "ruff format ",
+        "py-lint": "ruff check .",
+        "py-lint-files": "ruff check ",
+        "py-lint-files-exit-zero": "ruff check --exit-zero ",
+        "py-lint-fix": "ruff check --fix .",
+        "py-lint-fix-files": "ruff check --fix ",
```

Comment:
these are all the fun new ruff commands (surprisingly!)

Reply:
Shocking!